core/Pos/Util/Util.hs

```diff
@@ -325,7 +325,7 @@ type HasLens' s a = HasLens a s a
 
 -- | Version of 'lensOf' which is used when lens is to the same type
 -- as the tag.
-lensOf' :: forall s a. HasLens' s a => Lens' s a
+lensOf' :: forall a s. HasLens' s a => Lens' s a
```

Comment:
lol, it's not strange.
it's for convenient to avoid such code `lensOf' @_ @GenesisUtxo` and write instead `lensOf' @GenesisUtxo`